### ISSUE TITLE
Add command line functionality.

### ISSUE THREAD

**Issue by schmelz21** [cisagov/pe-reports#8]
# 💡 Summary #

Add in command line functionality that addresses cyhy database connection, data input and data output arguments.

## Motivation and context ##

Critical feature to ensure a secure and proper distribution of P&E Reports.

## Implementation notes ##

Intended usage should follow:

```
"""A tool for creating Posture & Exposure reports.

Usage:
    pe-reports REPORT_DATE DATA_DIRECTORY OUTPUT_DIRECTORY [--db-creds-file=FILENAME] [--log-level=LEVEL]

Arguments:
  REPORT_DATE                   Date of the report, format YYYY-MM-DD.
  DATA_DIRECTORY                The directory where the excel data files are located.
                                Organized by owner.
  OUTPUT_DIRECTORY              The directory where the final PDF reports should be saved.
  -c --db-creds-file=FILENAME   A YAML file containing the Cyber
                                Hygiene database credentials.
                                [default: /secrets/database_creds.yml]

Options:
  -h --help                     Show this message.
  -v --version                  Show version information.
  --log-level=LEVEL             If specified, then the log level will be set to
                                the specified value.  Valid values are "debug", "info",
                                "warning", "error", and "critical". [default: info]
"""
```

## Acceptance criteria ##

How do we know when this work is done?

- [x] Successfully generate P&E Reports from command line interface.
- [x] DATA_DIRECTORY input data file locations are not hard-coded.
- [x] OUTPUT_DIRECTORY is created when scripts are run.
- [x] Proper Exceptions are made if -db-creds-file is not available or fails.
- [x] log-level option type is added.

**Comment by schmelz21** [comment]
Current Branch - https://github.com/cisagov/pe-reports/tree/AL-working-v1.0 is running as intended, but needs work to comply to Acceptance Criteria and formal PR review. 

usage:
`pe-reports REPORT_DATE DATA_DIRECTORY OUTPUT_DIRECTORY [--db-creds-file=FILENAME] [--log-level=LEVEL]`

To avoid Matplotlib INFO logs which can be extensive set` --log-level=warning` or anything not "`info`" pr default.

**Comment by stewartl97** [comment]
@schmelz-ctr let's talk about this issue today.  I see two objectives (documentation and command line changes), and think they should be broken out, but I want to confirm that with you first.

**Comment by schmelz21** [comment]
This Issue can be closed on successful merge with https://github.com/cisagov/pe-reports/pull/33.
TODOS# in code has been removed on this commit 89c389d

**Comment by schmelz21** [comment]
> This Issue can be closed on successful merge with #33.
> TODOS# in code has been removed on this commit [89c389d](https://github.com/cisagov/pe-reports/commit/89c389d2f3a8a42fdabd077509fa2f1629542705)

Alternatively this task should be applied to `develop` branch too as it is now priority. PR - https://github.com/cisagov/pe-reports/pull/48
